keyboards/handwired/jscotto/scottoalp/keymaps/default/keymap.c

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2022 Joe Scotto
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+// Tap dance declarations
+enum {
+    TD_LCTL_ESC_SPOTLIGHT_EMOJI,
+    TD_LALT_ESC_WINDOWS_EMOJI
+};
+
+uint16_t get_tapping_term(uint16_t keycode, keyrecord_t *record) {
```

Comment:
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> default (and via) keymaps should be “pristine”
> * bare minimum to be used as a “clean slate” for another user to develop their own user-specific keymap

Reply:
Keymap is intentional as the default since the layout needs it.